kernel/selinux/sepolicy.c

```diff
@@ -9,6 +9,15 @@
 
 #define KSU_SUPPORT_ADD_TYPE
 
+/* Adapt to Huawei HISI kernel without affecting other kernels ,Huawei Hisi Kernel EBITMAP Enable or Disable Flag , From ss/ebitmap.h */
+#ifdef HISI_SELINUX_EBITMAP_RO &&LINUX_VERSION_CODE >=                         \
+        KERNEL_VERSION(4, 9, 0) ||                                             \
```

Comment:
should this be `||` or `&&`?

Reply:
Maybe I should express it this way:
`#ifdef HISI_SELINUX_EBITMAP_RO && (LINUX_VERSION_CODE >=                         \
        KERNEL_VERSION(4, 9, 0) ||                                             \
    LINUX_VERSION_CODE <= KERNEL_VERSION(4, 9, 250))`

Reply:
`(LINUX_VERSION_CODE >=                         \ KERNEL_VERSION(4, 9, 0) ||                                             \ LINUX_VERSION_CODE <= KERNEL_VERSION(4, 9, 250)` this means all version includes 1.0.0-6.2.0

Reply:
> `(LINUX_VERSION_CODE >= \ KERNEL_VERSION(4, 9, 0) || \ LINUX_VERSION_CODE <= KERNEL_VERSION(4, 9, 250)` this means all version includes 1.0.0-6.2.0

Yes,I get wrong idea.Sorry.